### PR TITLE
Dashboard view with styling

### PR DESCRIPTION
**Pull request by NickSaglioni**
# Description

Adding the dashboard page with all required components.

Jira task: [FSADT2-270](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-270)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit testing

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/components/LeftPanel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { SideNav, SideNavItems, SideNavLink } from '@carbon/react';
+
+import * as Icons from '@carbon/icons-react';
+
+const LeftPanel = ({ listItems }: any) => (
+  <SideNav isFixedNav expanded isChildOfHeader aria-label="Side navigation">
+    <SideNavItems>
+      {listItems.map((item: any) => (
```
Comment (RMCampos) [inline src/components/LeftPanel/index.tsx:10]: I think we can define an Interface here.

File: src/components/LeftPanel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div key={item.name}>
+          <SideNavLink key={item.name} title={item.name} disabled>
+            {item.name}
+          </SideNavLink>
+          {item.items.map((subItem: any) => {
```
Comment (RMCampos) [inline src/components/LeftPanel/index.tsx:15]: I think we can define an Interface here.

File: src/components/LeftPanel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { SideNav, SideNavItems, SideNavLink } from '@carbon/react';
+
+import * as Icons from '@carbon/icons-react';
```
Comment (RMCampos) [inline src/components/LeftPanel/index.tsx:5]: I'm not sure if this is a good idea. I would say that is clearer if we import only the icons that we're going to use.

File: src/types/Activity.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+type Activity = {
+    type: string;
+    status: number;
+    request_id: string;
+    created_at: string;
+    last_viewed: string;
```
Comment (RMCampos) [inline src/types/Activity.ts:6]: I think here there are 4 spaces instead of 2. Take a look on the lint rules.

**Review by RMCampos — CHANGES_REQUESTED** [review]

File: src/components/BCHeader/index.tsx [hunk 5]
```diff
@@ -25,8 +25,8 @@ const BCHeader = () => {
   return (
     <Theme theme="g100">
       <Header aria-label="BC Gov's NR Sample App" data-testid="header">
-        <Link to="/home" className="header-link" data-testid="header-name">
-          BC Gov&apos;s NR Sample App
+        <Link to="/dashboard" className="header-link" data-testid="header-name">
+          SPAR Seed Planning and Registry System
```
[the remark this answers is not in the document]
Comment (mgaseta) [reply src/components/BCHeader/index.tsx:29]: Fixed! Thanks for spotting that!

**Review by RMCampos — APPROVED** [review]
Great Job Matheus! 🥳

**Pull request merged**